recipe/meta.yaml

```diff
@@ -43,18 +43,16 @@ outputs:
         - bc
         - swig >=3.0.7
       host:
-        - blas 1.1 openblas
-        - openblas
+        - mkl-devel {{ mkl }}  # [blas_impl == "mkl"]
+        - openblas {{ openblas }}  # [blas_impl != "mkl"]
         - gsl
         - zlib
-        - fftw
+        - fftw  # [blas_impl != "mkl"]
```

Comment:
I believe that `fftw` remains necessary even in MKL builds because not all of LAL can yet select between the two. I have tested your recipe from the fork and it fails on 6.19.0 (as we expect).

I've tested removing `fftw` [from my hacked recipe that inspired your PR](https://git.ligo.org/thomas.downes/lal-mkl-feedstock) and it fails (on Linux) at

```
  CC       SkymapTest.o
SkymapTest.c:11:19: fatal error: fftw3.h: No such file or directory
 #include <fftw3.h>
```

@skymoo what are your plans to release 6.19.1? I'd like the answer to be "soon" so we can merge this and claim some pre-O3 MKL success, both for conda and other approaches. I realize that LAL impacts a lot of other things, but source tarballs would be sufficient for the purposes of conda.

@duncanmmacleod I appreciate your helping us do this right!

Reply:
@tpdownes, the `SkymapTest.c` source file includes `fftw3.h` but doesn't actually use it, see [here](https://git.ligo.org/lscsoft/lalsuite/merge_requests/615) for @skymoo's patch to fix that.

Reply:
I'm in the middle of rolling a 6.19.1 at the moment, the CI is running as once that's done I need to tag and roll the tarball. Probably a job for tomorrow as it's getting late here.